exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SchemaUtils.kt

```diff
@@ -313,7 +320,12 @@ object SchemaUtils {
                 val dataTypeProvider = currentDialect.dataTypeProvider
                 val redoColumns = existingTableColumns.mapValues { (col, existingCol) ->
                     val columnType = col.columnType
-                    val incorrectNullability = existingCol.nullable != columnType.nullable
+                    val colNullable = if (col.dbDefaultValue?.let { currentDialect.isAllowedAsColumnDefault(it) } == false) {
```

Comment:
Actually, I can not understand this part at the current moment >_<

Does it mean, that user can create a column like `<column>.default(<default-value>)`, and pass into the `default()` function something that could not be used as a default value, and in this case we create nullable column instead of throwing (received from database) sql exception?